src/main/java/com/powsybl/openloadflow/equations/Equation.java

```diff
@@ -111,6 +111,9 @@ public Equation addTerms(List<EquationTerm> terms) {
 
     private static double getBusTargetV(LfBus bus) {
         Objects.requireNonNull(bus);
+        if (bus.isDiscreteVoltageControlled()) {
```

Comment:
Probably to be done on lf network side but we should throw en exception in case of inconsistent target with transformer voltage control and generation voltage control on same bus.